dcat/index.html

```diff
@@ -2713,6 +2713,7 @@ <h3>Class: Dataset Series</h3>
             <tr><th>RDF Class:</th><td><a href="http://www.w3.org/ns/dcat#DatasetSeries"><code>dcat:DatasetSeries</code></a></td></tr>
             <tr><th class="prop">Definition:</th><td>A collection of datasets that are published separately, but share some common characteristics that groups them. <!--, and could also be made available as a single dataset.--></td></tr>
             <tr><th class="prop">Sub-class of:</th><td><a href="#Class:Dataset"><code>dcat:Dataset</code></a></td></tr>
+            <tr><th class="prop">Usage note:</th><td> Child datasets in dataset series SHOULD be represented as <code>dcat:Dataset</code>.<!--The use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.--></td></tr>
```

Comment:
@riccardoAlbertoni , I wonder whether we'd rather drop altogether the usage note. The fact that members of a series are instances of `dcat:Dataset` is already made explicit in the definition of this property, so there's no benefit in restating it in the usage note.